packages/nimble-components/src/table-column/icon/group-header-view/styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    bodyFont,
+    bodyFontColor,
+    smallPadding
+} from '../../../theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('inline-flex')}
+
+    :host {
+        gap: ${smallPadding};
+        align-items: center;
+    }
+
+    .no-shrink {
```

Comment:
We apply it to both icons and spinners in our templates. I kind of like that the name documents the purpose for having the class, rather than putting a seemingly redundant `icon-content` class on our icons and spinners. Since you don't feel strongly, I'd prefer to leave it.